app/src/main/java/org/fossasia/pslab/adapters/PerformExperimentAdapter.java

```diff
@@ -176,6 +181,8 @@ public Fragment getItem(int position) {
                         return InvertingOpAmpExperiment.newInstance();
                     case "Non Inverting Op-Amp":
                         return NonInvertingOpAmpExperiment.newInstance();
+                    case "RC Phase Shift":
+                        return RCPhaseShiftExperiment.newInstance();
                     case "Precision Rectifier":
```

Comment:
`Summing Junction` is there in `case 0`. Then it should not be there as it has nothing to do in this PR.